src/components/Progress/Progress.scss

```diff
@@ -7,6 +7,14 @@ $progress-m-height: 20px;
 $progress-s-height: 10px;
 $progress-xs-height: 4px;
 
+:root {
+    --flow-direction: 1;
+}
+
+[dir='rtl'] {
+    --flow-direction: -1;
+}
```

Comment:
Please you local variable here, with "private" naming:
```scss
.progress {
  --_--flow-direction: 1;
  
  [dir='rtl'] & {
    --_--flow-direction: -1;
  }
}
```

Reply:
Tnx, i changed it.